app/models/test_track/ab_configuration.rb

```diff
@@ -9,7 +9,10 @@ def initialize(opts)
 
     @true_variant = true_variant.to_s if true_variant
 
-    raise ArgumentError, "unknown split: #{split_name}" if @split_registry && !split
+    if @split_registry && !split
+      raise ArgumentError, "unknown split: #{split_name}. " \
```

Comment:
Nit, but a little awkward to have a trailing space if you aren't showing more stuff. Put that in the conditional text I think?